src/views/List.jsx

```diff
@@ -1,17 +1,18 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	console.log('data', data);
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item) => (
+					<div key={item.id}>
+						<ListItem name={item.name} />
+					</div>
```

Comment:
This works great and nice to see the use of the `key` prop. Since this is an [unordered list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul), `<ul />`, it should only contain [list items](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li), `<li />`.

The `<ListItem />` component renders an `<li />` so we're covered there so the `<div />` can be removed.

```suggestion
					<ListItem name={item.name} key={item.id} />
```